testament/specs.nim

```diff
@@ -265,6 +273,10 @@ proc isCurrentBatch*(testamentData: TestamentData; filename: string): bool =
 
 proc parseSpec*(filename: string): TSpec =
   result.file = filename
+  if retryContainer.retry and retryContainer.names.anyIt(it in result.file):
```

Comment:
So here is an idea, that you'll love.... Ok maybe love is a strong word, but if it's possible to store the full log, we could see which tests are newly discovered and need to be run, or only store successes?

So some failures, add tests, retry runs failure and new.